xsl/xccdf-guide-impl.xsl

```diff
@@ -351,6 +351,11 @@ Authors:
 
                 <td style="padding-left: {$indent * 19}px">
                     <h3>
+                        <xsl:if test="$indent=1 or $indent=2">
+                            <xsl:attribute name="id">
```

Comment:
Why use title as id attribute when the id attribute would be much more suitable? It requires no translation.